components/GetstartedCard/index.tsx

```diff
@@ -63,7 +63,7 @@ export const PlainCard = ({ desc, title, route }: MyProps) => {
 // the difference the 1st function with the rest is it has an icon on the top
 
 // 1 Start integrating
-export default function GetStarted(): JSX.Element {
+export default function GetStarted(props: any): JSX.Element {
   return (
     <section
       className={classNames(
```

Comment:
Is there a reason for the props? Don't see where it's being used

Reply:
Ooo no worries I'll take 'em off.